app/model/SectionAudit.scala

```diff
@@ -20,6 +21,16 @@ case class SectionAudit(
                      sectionSummary: SectionSummary
                    ) {
   def toItem = Item.fromJSON(Json.toJson(this).toString())
+
+  def asAuditingThrift = Notification(
+    app = App.TagManager,
+    operation = operation,
+    userEmail = s"${user.replace(" ", ".").toLowerCase}@guardian.co.uk",
```

Comment:
Unfortunately not at the moment (I've added a flaw friday item for me to switch to using that rather than name to identify users with our job system)